src/TypeSpec.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -142,6 +144,13 @@ function getCSharpInputTypeKindByIntrinsicModelName(
             return InputTypeKind.Float32;
         case "float64":
             return InputTypeKind.Float64;
+        case "uri":
```

Comment:
I think this fixes other issues?

Reply:
Yes, to upgrade the new version of tcgc which contains getAllModels, some gaps needs to fill.